src/Python/Meshes/SolidClip.py

```diff
@@ -0,0 +1,61 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import vtk
+
+
+def main():
+    # Create a superquadric
+    superquadric_source = vtk.vtkSuperquadricSource()
+    superquadric_source.SetPhiRoundness(3.1)
+    superquadric_source.SetThetaRoundness(2.2)
+
+    # Define a clipping plane
+    clip_plane = vtk.vtkPlane()
+    clip_plane.SetNormal(1.0, -1.0, -1.0)
+    clip_plane.SetOrigin(0.0, 0.0, 0.0)
+
+    # Clip the source with the plane
+    clipper = vtk.vtkClipPolyData()
+    clipper.SetInputConnection(superquadric_source.GetOutputPort())
+    clipper.SetClipFunction(clip_plane)
+
+    # Create a mapper and actor
+    superquadric_mapper = vtk.vtkPolyDataMapper()
+    superquadric_mapper.SetInputConnection(clipper.GetOutputPort())
+
+    superquadric_actor = vtk.vtkActor()
+    superquadric_actor.SetMapper(superquadric_mapper)
+
+    # Create a property to be used for the back faces. Turn off all
+    # shading by specifying 0 weights for specular and diffuse. Max the
+    # ambient.
+    back_faces = vtk.vtkProperty()
+    back_faces.SetSpecular(0.0)
+    back_faces.SetDiffuse(0.0)
+    back_faces.SetAmbient(1.0)
+    back_faces.SetAmbientColor(1.0000, 0.3882, 0.2784)
+
```

Comment:
In the "old days" this was acceptable. Can you please use vtkNamedColors instead.

Reply:
Shall I directly use this:
```
from vtk.util.colors import tomato
(...)
back_faces..SetDiffuseColor(tomato)
```
or do you prefer me to do
```
colors = vtk.vtkNamedColors()
back_faces.SetAmbientColor(colors.GetColor3d("tomato").GetData())
```
?

Reply:
Olease use vtkNamedColors, in the case of Python, you don't need the ` ,GetData()`

Reply:
Done in b6ca8de.